### DISCUSSION TITLE
Favicon / Icon Hash

### DISCUSSION THREAD

**Discussion by idlefire** [projectdiscovery/httpx#458] [Ideas] [answered]
### Please describe your feature request:

MATCHERS add icon hash
### Describe the use case of this feature:

website finger

**Comment by ehsandeep** [comment]
Hi @idlefire,

Please add more details around this feature request, based on current information I'm not sure how the feature is usable with httpx?

**Comment by idlefire** [comment]
Web sites have Icon, which can be used to generate Icon hash for site fingerprint matching, similar to shodan, which can quickly filter out specific applications. [Like this](https://medium.com/@Asm0d3us/weaponizing-favicon-ico-for-bugbounties-osint-and-what-not-ace3c214e139)

**Comment by ehsandeep** [comment, marked as answer]
This is now supported with release of https://github.com/projectdiscovery/httpx/releases/tag/v1.1.5
